lms/envs/aws.py

```diff
@@ -133,6 +133,10 @@
 MEDIA_ROOT = ENV_TOKENS.get('MEDIA_ROOT', MEDIA_ROOT)
 MEDIA_URL = ENV_TOKENS.get('MEDIA_URL', MEDIA_URL)
 
+# GITHUB_REPO_ROOT is the base directory
+# for course data
+GITHUB_REPO_ROOT = ENV_TOKENS.get('GITHUB_REPO_ROOT', GITHUB_REPO_ROOT)
```

Comment:
Why was this added (as opposed to `COURSES_ROOT`)? That's a genuine question -- I don't understand the relationship, or where this was defined before (I notice that it existed in code that was moved?).